_posts/2017-08-16-Refactoring-with-ReactJS-testing.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+authors:
+ - ilyaGotfryd
+ - aliOrlando
+date: 2017-08-16 01:30:00 -0500
+title: "Testing and Infrastructure: Refactoring a Legacy Application in React - Part 3"
+test_tools:
+ - image_path: /images/karma_icon.png
+   tool_name: Karma
+   tool_description: The main goal for Karma is to bring a productive testing environment to developers. The environment being one where they don't have to set up loads of configurations, but rather a place where developers can just write the code and get instant feedback from their tests.
```

Comment:
I like that you include why we chose Karma, but I'm not sure that this really tells me what the tool is.

Reply:
To be frank, I jacked their phrasing. I'll put a bit more effort into it.